content/community/conferences.md

```diff
@@ -64,6 +50,21 @@ Oct 5 - 7, 2023. In-person in Goa, India (hybrid event) + Oct 3 2023 - remote da
 
 ## Past Conferences {#past-conferences}
 
+### React Global Online Summit 22.2 by Geekle {#react-global-v2-2022}
```

Comment:
Can you re arrange the same in the reverse order? so that the last conference is the first in past conferences? older ones are that way?

Reply:
Hi @harish-sethuraman good shout, updated now. 

Maybe will raise another pr where code can handle the ordering so this doesn't fall out of date on its own ?